pmd-dart/src/main/antlr4/net/sourceforge/pmd/lang/dart/antlr4/Dart2.g4

```diff
@@ -665,17 +665,17 @@ statements
   : statement*
   ;
 statement
-  : label* nonLabledStatment
+  : label* nonLabelledStatement
```

Comment:
Wouldn't these changes in Dart2.g4 change the AST API? Hm... ok, we only support CPD for Dart, so it's fine.